mautic/templates/deployment.yaml

```diff
@@ -37,8 +37,6 @@ spec:
               value: {{ .Values.database.password | quote }}
             - name: MAUTIC_RUN_CRON_JOBS
               value: {{ .Values.runCronJobs | quote }}
-            - name: MAUTIC_TRUSTED_PROXIES
```

Comment:
Why you removed the trusde proxies? =\

Reply:
Due to changes in the 3rd version of the mautic, the "MAUTIC_TRUSTED_PROXIES" variable is no longer used and its definition causes errors

Reply:
So how does we need to setup trusted proxies in version 3?

Reply:
https://github.com/mautic/docker-mautic/blob/v3-beta/common/makeconfig.php#L34

Here is where it is used in Docker v3 =\

Reply:
We can set trusted proxies up later using interface or manually in the local.php config file

Reply:
Trusted proxy feature part of the Docker image =\ it is MUST be in the chart as well.

In kubernetes by default our service will be after the proxy.
So in kubernetes Mautic instance the trusted proxy must be configured on installation.